src/NECompletion/CompletionContext.class.st

```diff
@@ -108,10 +110,7 @@ CompletionContext >> hasMessage [
 
 { #category : #entries }
 CompletionContext >> initEntries [
-	| suggestionsList sorter |
-	sorter := self class sorterClass new context: self.
-	suggestionsList := sorter sortCompletionList: (node completionEntries: position).
-	^ suggestionsList
+	^ sorter sortCompletionList: (node completionEntries: position) asOrderedCollection
```

Comment:
Did you think to avoid sorter classes at all? 
I think they can be easily replaced by SortFunction instances. In this case the code would look like:
```Smalltalk
(node completionEntries) sorted: sortFunction
``` 
For simple alphabetic order we would have
```Smalltalk
SortFunction default
```
And of reversed order:
```
SortFunction default reversed
```

Reply:
The idea was that people should be able to create more sorters, and those showing up in the Settings automatically

Reply:
@dionisiydk This implementation was specifically created so it would be extendable and could include more sophisticated sorting strategies all part of this one tool.